src/course_bot/essay.clj

```diff
@@ -38,8 +59,32 @@
     :set-of-reviews "Есть комплект ревью на: "
     :essays " эссе."
     :assignment-error "ОШИБКА: почему-то не удается найти задание!"
-    :assignment-count "Количество заданий:"
-    :assignment-examples "Примеры: "}}})
+    :assignment-count "Количество заданий: "
+    :assignment-examples "Примеры: "
+    :first-essay-best "Первое эссе -- лучшее.\n\n"
+    :rank-1 "Ранг: %d, "
+    :essay-number-in-the-list-1 "Номер эссе в списке: #%d, "
+    :your-review-1 "Ваше ревью: %s "
+    :few-words-from-the-essay "\n(несколько слов из эссе: "
```

Comment:
not used?

Reply:
> Only essay?

Other modules in progress

Reply:
> not used?

Yeah, I'll clean it, because it was later combined into one text